pkg/broker/work_engine_test.go

```diff
@@ -0,0 +1,106 @@
+package broker
+
+import (
+	"fmt"
+	"testing"
+
+	ft "github.com/openshift/ansible-service-broker/pkg/fusortest"
+)
+
+var engine *WorkEngine
+
+func init() {
+	engine = NewWorkEngine(10)
+}
+
+type mockSubscriber struct {
+	buffer <-chan WorkMsg
+	called bool
+}
+
+func (ms *mockSubscriber) Subscribe(buffer <-chan WorkMsg) {
+	ms.buffer = buffer
+	ms.called = true
+}
+
+type mockMsg struct {
+	msg string
+}
+
+func (mm mockMsg) Render() string {
+	return mm.msg
+}
+
+type mockWorker struct {
+	called bool
+}
+
+func (mw mockWorker) Run(token string, buffer chan<- WorkMsg) {
+	fmt.Println("xxxxxxxxxxxxxxxxxxxxx run called")
+	mw.called = true
```

Comment:
The code where this gets used is commented out.  Does ```TestStartNewJob``` need to be uncommented or should we remove the mockWorker struct?

Reply:
I'll clean that up. I think I wsa in the middle of creating a test that didn't quite work because I needed to mock something else.